docs/src/main/sphinx/connector/hive.rst

```diff
@@ -666,6 +666,7 @@ features:
   * :ref:`sql-views-management`
 
 * :ref:`sql-security-operations`, see also :ref:`hive-sql-standard-based-authorization`
+* :ref:`sql-transactions`
```

Comment:
cc: @losipiuk confirm if transactions can be used with all types of Hive tables?